### PR TITLE
fix package_has_file function for .conda format (use cph)

### PR DESCRIPTION
**Pull request by msarahan**

### CODE REVIEW

File: conda_build/index.py [hunk 1]
```diff
@@ -1430,48 +1233,22 @@ def _add_extra_path(extra_paths, path):
         _add_extra_path(extra_paths, join(subdir_path, REPODATA_JSON_FN + '.bz2'))
         _add_extra_path(extra_paths, join(subdir_path, REPODATA_FROM_PKGS_JSON_FN))
         _add_extra_path(extra_paths, join(subdir_path, REPODATA_FROM_PKGS_JSON_FN + '.bz2'))
-        _add_extra_path(extra_paths, join(subdir_path, "repodata2.json"))
+        # _add_extra_path(extra_paths, join(subdir_path, "repodata2.json"))
```
Comment (soapy1) [inline conda_build/index.py]: why is repodata2.json getting removed?

Comment (msarahan) [reply conda_build/index.py]: It has never been used. We can bring it back when we do namespaces. I think enough about the repodata will change with sharding that this stuff would need major rework anyway.

**Comment by soapy1** [comment]
Tried this out, oh my, so much faster! 🚀

**Comment by msarahan** [comment]
This seems mostly good to go.  We finally have a fully passing test suite with conda 4.7.  On Windows, we still have some permissions errors.  I think these are related to problems we've been seeing lately elsewhere (https://github.com/conda-forge/conda-feedstock/pull/95)

I think we should hold off on the next release until those are solved.  I'll take a look tomorrow.

**Pull request merged**